content/blog/glc-scim-api-integration-with-okta-scim-adapter-1.md

```diff
@@ -0,0 +1,5933 @@
+---
+title: GLC SCIM API Integration with Okta SCIM Adapter
+date: 2024-05-13T20:55:22.189Z
+author: Namik Rujnic - Cloud Operations
+authorimage: /img/Avatar1.svg
+disable: false
+---
+<!--\\[if gte mso 9]><xml>
```

Comment:
Hi Namik,

You probably do a cut and paste from a MS-Word document to the CMS editor without a Plain text copy (CTRL+SHIFT+V)  to not get unwanted XML fragment code syntaxes.
Copy/paste from MS-Word without Plain Text generate XML fragment code syntaxes in your blog post.
If you go to the CMS editor and enable the "Markdown mode", you will probably see the XML syntaxes

--> Please remove the XML syntaxes from your post.